src/RpcServer/RpcServer.SmartContract.cs

```diff
@@ -78,13 +78,7 @@ private JObject GetInvokeResult(byte[] script, Signers signers = null)
             json["exception"] = GetExceptionMessage(engine.FaultException);
             try
             {
-                int max = settings.MaxResultItems;
-                string warning = null;
-                json["stack"] = new JArray(engine.ResultStack.Select(p => ToJson(p, ref max, ref warning)).Where(u => u is not null));
-                if (!string.IsNullOrEmpty(warning))
-                {
-                    json["warning"] = warning;
-                }
+                json["stack"] = new JArray(engine.ResultStack.Select(p => ToJson(p, settings.MaxResultItems)));
```

Comment:
Now is not a `MaxResultItems` it's a `MaxIteratorResultItems`

Reply:
Yes. We should rename it.